mongo/integration/mtest/setup.go

```diff
@@ -300,19 +311,50 @@ func addCompressors(uri string) string {
 	return addOptions(uri, "compressors=", comp)
 }
 
+func addServerlessAuthCredentials(uri string) (string, error) {
+	if os.Getenv("SERVERLESS") != "serverless" {
+		return uri, nil
+	}
+	user := os.Getenv("SERVERLESS_ATLAS_USER")
+	if user == "" {
+		return "", fmt.Errorf("serverless expects SERVERLESS_ATLAS_USER to be set")
+	}
+	password := os.Getenv("SERVERLESS_ATLAS_PASSWORD")
+	if user == "" || password == "" {
```

Comment:
The `user == ""` check is redundant with the previous user check and can be removed.

Reply:
Good point; removed.